sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -364,13 +364,15 @@ public partial class ServiceBusReceiver : System.IAsyncDisposable
         public virtual System.Threading.Tasks.Task DeadLetterMessageAsync(Azure.Messaging.ServiceBus.ServiceBusReceivedMessage message, System.Collections.Generic.IDictionary<string, object> propertiesToModify = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task DeadLetterMessageAsync(Azure.Messaging.ServiceBus.ServiceBusReceivedMessage message, string deadLetterReason, string deadLetterErrorDescription = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task DeferMessageAsync(Azure.Messaging.ServiceBus.ServiceBusReceivedMessage message, System.Collections.Generic.IDictionary<string, object> propertiesToModify = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<int> DeleteMessagesAsync(int messageCount, System.DateTimeOffset? beforeEnqueueTimeUtc = default(System.DateTimeOffset?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Should this be a `long`? I know the SequenceNumber is a long though I don't know if it is realistic to have over 2 billion messages in a queue.

Reply:
Should it be "enqueued" rather than "enqueue"?

Reply:
The service team has this capped at 4,000 and specified an int, so I don't see us as needing to move to long.  As it is, the arbitrary service cap makes `int` confusing because `int.MaxValue` gets rejected.

Reply:
> Should it be "enqueued" rather than "enqueue"?

There was a bunch of discussion on this name at the archboard, and "enqueue" won out.  The languages are aligned on it, so I'm going to leave it for now.   We can revisit before GA if we want to bring it back up for discussion.